pkg/custom/client.go

```diff
@@ -31,7 +31,7 @@ func NewClient(c Connection) (*Client, error) {
 
 func (cli *Client) Run() (common.Result, error) {
 	var result common.Result
-	req := &schemav1.AnalyzerRunRequest{}
+	req := &schemav1.RunRequest{}
```

Comment:
Shouldn't we rename this to `RunAnalyzeRequest` ?

Reply:
I could update this in an update?